issuegenerator/main_test.go

```diff
@@ -108,3 +108,42 @@ func TestIngestArtifacts(t *testing.T) {
 	require.Equal(t, expectedTestResults, rg.testSuites)
 
 }
+
+func TestTrimPath(t *testing.T) {
+
+	tests := []struct {
+		name           string
+		owner          string
+		repo           string
+		module         string
+		expectedModule string
+	}{
+		{
+			name:           "Test contrib host metrics integration path with https prefix",
+			owner:          "open-telemetry",
+			repo:           "opentelemetry-collector-contrib",
+			module:         "https://github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/integration_test.go",
+			expectedModule: "receiver/hostmetricsreceiver/integration_test.go",
```

Comment:
The module that we get from automation is actually the [module name as written in the go.mod file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/df65f76b84de0683b892ee5f32e72f8781d44f0a/receiver/hostmetricsreceiver/go.mod#L1), not the test file name. 

But not really important, the trimming is what's being tested and it works 👍

Reply:
Thanks for the information. I've removed the prefix check and testing, we can keep it simple for now and change as needed.